packages/account/src/Components/self-exclusion/__tests__/self-exclusion-confirm-page.spec.tsx

```diff
@@ -13,7 +13,7 @@ jest.mock('../self-exclusion-confirm-limits', () => {
 const mockUseFormikContext: FormikValues = jest.spyOn(formik, 'useFormikContext');
 
```

Comment:
no types for mock context?

Reply:
Are u refering to Ln 16? The type is added `TSelfExclusionContext`